data/xml/2023.insights.xml

```diff
@@ -83,10 +83,8 @@
       <author><first>Louis</first><last>Martin</last><affiliation>Meta AI</affiliation></author>
       <pages>11-18</pages>
       <abstract>Embedding representations of text are useful for downstream natural language processing tasks. Several universal sentence representation methods have been proposed with a particular focus on self-supervised pre-training approaches to leverage the vast quantities of unlabelled data. However, there are two challenges for generating rich embedding representations for a new document. 1) The latest rich embedding generators are based on very large costly transformer-based architectures. 2) The rich embedding representation of a new document is limited to only the information provided without access to any explicit contextual and temporal information that could potentially further enrich the representation. We propose efficient retrieval-augmented text embeddings (ERATE) that tackles the first issue and offers a method to tackle the second issue. To the best of our knowledge, we are the first to incorporate retrieval to general purpose embeddings as a new paradigm, which we apply to the semantic similarity tasks of SentEval. Despite not reaching state-of-the-art performance, ERATE offers key insights that encourages future work into investigating the potential of retrieval-based embeddings.</abstract>
-      <url hash="40e8e7da">2023.insights-1.2</url>
+      <url hash="8270255e">2023.insights-1.2</url>
       <bibkey>raina-etal-2023-erate</bibkey>
-      <video href="2023.insights-1.2.mp4"/>
-      <doi>10.18653/v1/2023.insights-1.2</doi>
     </paper>
```

Comment:
Here the video and doi tags have been removed.

Reply:
Done.